docs/pages/enroll-resources/machine-id/workload-identity/federation.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: SPIFFE Federation
+description: An overview of the Teleport Workload Identity SPIFFE Federation feature.
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. This means that some
+features may be missing. We're actively looking for design partners to help us
+shape the future of Workload Identity and would love to
+[hear your feedback](mailto:product@goteleport.com).
+</Admonition>
+
+Federation is the ability for a trust relationship to be established between
+two trust domains, allowing workloads in one trust domain to validate the
+identities of workloads in the other trust domain.
+
+SPIFFE sets out a standard for federation that allows trust domains managed by
+different implementations to federate with one another. For example, a trust
+domain managed by Teleport Workload Identity could federate with a trust domain
+managed by SPIRE.
+
+The SPIFFE Federation process relies on the exchange of trust bundles between
+the trust domains. These trust bundles contain the certificates and public keys
+of the trust domain's issuer.
+
+Federation relationships are "one way" meaning that workloads within one trust
+domain can validate the identities of workloads in the other trust domain, but,
+workloads in the other trust domain cannot validate the identities of workloads
+in the first trust domain. It is typical to therefore establish the federation
+relationship in both directions.
+
+<Admonition type="warning" title="Teleport Enterprise Required">
+A valid Teleport Enterprise license is required to use the federation features
+of Teleport Workload Identity.
+</Admonition>
+
+## Federation to Teleport Workload Identity
+
+This section discusses how to configure another trust domain to trust a Teleport
+Workload Identity hosted trust domain.
+
+In order to configure a trust domain to trust Teleport Workload Identity, the
+control plane of the trust domain must be able to access the SPIFFE Bundle
```

Comment:
What does "control plane" refer to in this context? Is control plane a well-understood concept in SPIFFE?

Reply:
It's fairly general engineering term, it's whatever component is managing/maintaining the other trust domain. Which depending on what software they're using, is maybe a specific thing like our "Auth Service" or maybe something totally different.

 It's hard to generalise here, it'll be easier to be more specific when we write up some guides for federating with specific implementations but I don't have time for that at the moment and with the current greenfield nature of Workload ID, we're working closely with most customers using it, so I'm not convinced writing a bunch of specific how-to guides is worth my time atm.